### PR TITLE
fix(slider): update styles for disabled state on slider

### PR DESCRIPTION
**Pull request by j1mie**
Fixes disabled color values / focus behaviour for the slider component.

#### Changelog

**New**

- Adds value to `slider` config
- Adds disabled slider story to fractal
- Ignore jetbrains / idea files

**Changed**

- Places disabled modifier classes below main styles
- Prevents disabled slider thumb from resizing on focus / blur
- Prevents disabled slider track from changing color
- Uses disabled color values on text input when the slider component is disabled

**Note**
I avoided nesting SASS except for pseudo classes as I'd seen this elsewhere in the repo - let me know if this is the correct approach!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@ project.sublime-project
 project.sublime-workspace
 jsconfig.json
 .vscode
+.idea
```
Comment (asudoh) [inline .gitignore:29]: What this is for?

**Review by asudoh — COMMENTED** [review]
Hi @j1mie thank you for jumping in! CC @IBM/carbon-designers to see if such UI change makes sense.

File: .gitignore (hunk 1 above)
Comment (j1mie) [reply .gitignore:29]: This is to ignore JetBrains IDE files

**Review by aagonzales — COMMENTED** [review]
Can you change `label` and first value to use `$disabled-02`. In the system, we're making the labels disabled with the the component.

**Review by jnm2377 — APPROVED** [review]
Code looks good, other than the token change @aagonzales requested. Thanks for the PR! 🎉

**Comment by joshblack** [comment]
Just pushed up the updates I think you were requesting @aagonzales!

**Review by tw15egan — APPROVED** [review]
Looks good with the recent changes

**Pull request merged**